adapters/Facebook/FacebookAdapter/OpenBidding/GADFBRewardedRenderer.m

```diff
@@ -108,6 +111,11 @@ - (void)rewardedVideoAdDidLoad:(FBRewardedVideoAd *)rewardedVideoAd {
 }
 
 - (void)rewardedVideoAd:(FBRewardedVideoAd *)rewardedVideoAd didFailWithError:(NSError *)error {
+  if (_presentCalled) {
+    NSLog(@"Received a Facebook SDK error during presentation: $@", error.localizedDescription);
+    _adLoadCompletionHandler(nil, error);
```

Comment:
Can we drop rows 116-117 as we still send the error on 119?

Reply:
It was a code error, and it was corrected. Line 116 should be [_adEventDelegate didFailToPresentWithError:error];